rank/data/indices/works-with-archive-descriptions.ts

```diff
@@ -6,7 +6,6 @@ import { multilingualFields, shinglesFields } from './common'
 export default {
   mappings: {
     dynamic: 'strict',
-    type: "object",
```

Comment:
FYI - this is here because of stupic elastic4s and how we compare JSON queries. No fuss, just letting you know.

Reply:
`createTestIndex` was v unhappy when this line was included - any idea whether there's a way of satisfying both?